docs/source/isaura.rst

```diff
@@ -1,5 +1,209 @@
+
+
+.. _Hits-based selection:
+
+Event selection based on the contained hits
+:::::::::::::::::::::::::::::::::::::::::::
+
+
+First of all, it is mandatory to perform some selections concerning the number of hits that events contain, in order to be able to compute all the tracking information for each of them.
+  
+The first condition that all events must fulfill to be processed is to contain at least one hit. If that does not happen, the event will be rejected, which will be displayed in the table ``Filters/hits_select``.
+
+The next step within the algorithm consists in checking that the number of hits is lower than the value provided in the config file (``max_num_hits``). That argument was introduced because, when running *Paolina* algorithm after :doc:`penthesilea`, there were some events that comprise such large amount of hits that the tracking information extraction took ridiculously long. The following picture shows the number of *Penthesilea hits* (**hdst**) per event (with a different scale) for a typical 24h-long low-background run included in the NEXT-White double-beta analysis [#]_. High energy (trigger2) events usually contain around 200 Penthesilea hits (as right panel points out), while there are some of them with more than 10000 hits (illustrated in left panel).
+
+ .. image:: images/isaura/nhits_per_evt_r8571.jpg
+   :width: 1000
+
+The plot also shows that these events only  appear a few times within a 24h-long low-background run (around 0.05% of the total set of events). Rejecting this type of events is not a particularly worrysome issue: they would be thrown away in the posterior analysis, owing to the fact that none of them are exclusively contained inside the fiducial volume. The spatial distributions for one of these events is presented below (where the grey dashed lines illustrate the boundaries of the chamber).
+
+
+ .. image:: images/isaura/XY_Z_distributions_evt_many_hits.jpg
+   :width: 1000
+
+In any case, one can easily infer from the plots that these events are not physical. On the contrary, they seem to correspond to either some kind of flash occurring in the chamber (like a mini-spark) or some fail in the electronics (after the saturation of an alpha particle, for example). The ID of the events that are removed from the reconstruction chain because of this reason will be specified in the table ``Filters/topology_select``, in order to keep track of this information.
+
+..
+ Next step includes another (quite obvious) check: at least one hit inside the event must have a well-defined energy. If not, the event will be also rejected, since no topological information could be extracted.
+
+
+Finally, every event is also required to contain hits with well-defined energy. For instance, events with all hits outside the krypton correction map boundaries will be thrown away, since their energy cannot be corrected and their ``Ec`` variable (*corrected energy*) will be ``NaN``.
```

Comment:
Maybe mention here where is Ec included?

Reply:
Sorry, do you mean in which table is `Ec`? or at what stage is this parameter computed?

Reply:
I've rephrased the whole paragraph, trying to be clearer... Do you think it is better now?